examples/amp-list.amp.html

```diff
@@ -75,6 +83,24 @@
     </script>
   </amp-list>
 
+  <!-- ## Using a script template with custom mustache delimiters. -->
+  <amp-list width="auto" height="100" layout="fixed-height" src="comments.example.json">
+    <script type="text/plain" template="amp-mustache" custom-delimiters="#-,-#">
+      <table>
+        <tr>
+          <td>Comment:</td>
+          <td>#-content-#</td>
+        </tr>
+        #-#replies-#
```

Comment:
Iteration syntax with alternative custom delimiter is not that pleasant to look at. Maybe reconsider not using `#`?